presto-server-rpm/src/main/resources/dist/etc/init.d/presto

```diff
@@ -36,7 +36,7 @@ start () {
     if [ -z "$JAVA8_HOME" ]
     then
         echo "Warning: No value found for JAVA8_HOME. Default Java will be used."
-        sudo -u $SERVICE_USER /usr/lib/presto/bin/launcher start ${CONFIGURATION[@]}
+        sudo -u $SERVICE_USER /usr/lib/presto/bin/launcher start "${CONFIGURATION[@]}"
```

Comment:
Please correct if I am wrong, this was failing in case of $DATA_DIR containing a space?

Reply:
Yes. If any element in `$CONFIGURATION` array contained a whitespace.

Reply:
Thanks. Can you add this information to commit message.

Reply:
We use `"${CONFIGURATION[@]}"` like 10 times in the file already. I don't think it's necessary (or that this partiocular commit message would be findable)